config/routes.rb

```diff
@@ -350,12 +350,16 @@
       put '/how_to_verify' => 'how_to_verify#update'
       get '/document_capture' => 'document_capture#show'
       put '/document_capture' => 'document_capture#update'
+      get '/socure/document_capture' => 'socure/document_capture#show'
+      put '/socure/document_capture' => 'socure/document_capture#update'
```

Comment:
Socure only supports GET and POST for the redirect .... could also punt on this til [LG-14007](https://cm-jira.usa.gov/browse/LG-14007) and [LG-14008](https://cm-jira.usa.gov/browse/LG-14008)

Reply:
If this route is only for our side of things, would Socure's requirements have an impact, or is it only for requests being sent to their servers?

Reply:
@mitchellhenke  ... our side of things? pls elaborate
after the user has submitted their ID images and completed the CaptureApp experience, Socure needs somewhere to send them ... Socure will  redirect them  back to Login at POST '/socure/document_capture'

Reply:
Oh I see, I didn't understand that Socure was making inbound requests to us.